products/jbrowse-web/src/createPluginManager.ts

```diff
@@ -25,15 +25,15 @@ export function createPluginManager(
       plugin: new P(),
       metadata: { isCore: true },
     })),
-    ...model.runtimePlugins.map(({ plugin: P, definition }) => ({
+    ...(model.runtimePlugins ?? []).map(({ plugin: P, definition }) => ({
```

Comment:
this won't actually, in practice, ever be undefined, is that correct?

Reply:
`runtimePlugins` won't ever be undefined since this is only called after the session loader is "ready," but `sessionPlugins` might be undefined in e.g. the case of a blank session where session plugin loading is never done.